src/components/index.ts

```diff
@@ -1,7 +1,9 @@
 import GiftModule from './gift/gift.module';
 import GiftHistoryModule from './gift_history/gift_history.module';
+import ItemModule from './item/item.module'
 import UserModule from './user/user.module';
 
-const Components = [UserModule, GiftModule, GiftHistoryModule];
+
+const Components = [UserModule, GiftModule, GiftHistoryModule, ItemModule];
```

Comment:
ちょっと個人的な好みですが、`schema.prisma`の順番に倣ってこれでお願いします。

```suggestion
const Components = [UserModule, ItemModule, GiftModule, GiftHistoryModule];
```